Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version:5.2
+
+import PackageDescription
+
+let package = Package(
+    name: "Proton",
+    platforms: [
+        .iOS(.v13),
```

Comment:
Can you please change the supported version to iOS 10? I am considering dropping support for lower versions of iOS, but don't have a good reason as yet :)

Reply:
@rajdeep done, but to 11. can't lower the minimum version to anything below than iOS 11 (`SnapshotTesting` requires it).